src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1677,15 +1694,20 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
         // NOTE: can return null if the method has no body.
         internal static BoundBlock BindMethodBody(MethodSymbol method, TypeCompilationState compilationState, BindingDiagnosticBag diagnostics)
         {
-            return BindMethodBody(method, compilationState, diagnostics, nullableInitialState: null, includesFieldInitializers: false, out _, out _, out _);
+            return BindMethodBody(method, compilationState, diagnostics, nullableInitialState: null, reportNullableDiagnostics: false, includesFieldInitializers: false, out _, out _, out _);
```

Comment:
>false [](http://example.com/codeflow?start=128&length=5)

It is not clear why this is always the right value.

Reply:
> Yeah, I am not sure either.

Given the current call sites for the containing method, I think ```true``` will be a better default.

Reply:
ok